dask_sql/physical/rel/logical/sort.py

```diff
@@ -21,7 +20,13 @@ def convert(self, rel: "LogicalPlan", context: "dask_sql.Context") -> DataContai
         (dc,) = self.assert_inputs(rel, 1, context)
         df = dc.df
         cc = dc.column_container
-
+        sort_expressions = rel.sort().getCollation()
+        sort_columns = [
+            cc.get_backend_by_frontend_name(expr.column_name(rel))
+            for expr in sort_expressions
+        ]
+        sort_ascending = rel.sort().getAscending()
+        sort_null_first = rel.sort().getNullsFirst()
         # TODO: Commented out to pass flake8, will be fixed in sort PR
         # sort_collation = rel.getCollation().getFieldCollations()
         # sort_columns = [
```

Comment:
Do we want to leave in this commented out code?

Reply:
Good point, I'll remove